shared/general/container-ssh.md

```diff
@@ -43,35 +41,50 @@ $ balena ssh 192.168.1.23
 $ balena ssh <device-uuid>.local
 ```
 
-This should work without further configuration in the case of devices running a [development {{ $names.os.lower }} image][development-image], which allows passwordless root SSH access (and for this reason, should never be directly exposed to the public internet). In the case of a production {{ $names.os.lower }} image, an SSH key needs to be present in the config.json file of the device. More details about configuring SSH keys in config.json may be found [here][config-json-ssh].
-
-## Add an SSH key to {{ $names.cloud.lower }}
-
-To add an SSH key, go to the _Preferences_ page of {{ $names.cloud.lower }} and select the _SSH Keys_ tab.
-
-![SSH key preferences](/img/common/main_dashboard/eekVBTI.png)
-
-You may either import an existing SSH key from GitHub or manually enter the public SSH key of an existing SSH key on your development machine.
-
-If you do not have an existing key, you can follow [GitHub's documentation][github-ssh], skipping the step about adding the key to your GitHub account, and instead adding the key to your {{ $names.cloud.lower }} account.
+This should work without further configuration in the case of devices running a [development
+{{ $names.os.lower }} image][development-image], which allows unauthenticated `root` access
+(and for this reason, should never be directly exposed to the public internet). In the case of
```

Comment:
Replaced "passwordless root SSH access" with "unauthenticated `root` access" because "passwordless" can be interpreted as "authenticated against a public key".